src/handler.ts

```diff
@@ -19,16 +19,16 @@ interface LineDecorationText {
  * @returns repo and file part of URI.
  */
 export function getParamsFromUriPath(textDocumentURI: string): Params {
```

Comment:
I would recommend using the `URL` API first on this string to get the host+path and hash out.

Reply:
Will follow up on this. Issue tracking this here: https://github.com/sourcegraph/sourcegraph-sentry/issues/37